custom-set.json

```diff
@@ -261,279 +121,156 @@
       "cases": [
          {
-            "description": "difference of two empty sets",
+            "description": "difference of two empty sets is an empty set",
             "set1": [],
             "set2": [],
             "expected": []
          },
          {
-            "description": "difference of empty set and non-empty set",
+            "description": "difference of empty set and non-empty set is an empty set",
             "set1": [],
             "set2": [3, 2, 5],
             "expected": []
          },
          {
-            "description": "difference of non-empty set and empty set",
+            "description": "difference of a non-empty set and an empty set is the non-empty set",
             "set1": [1, 2, 3, 4],
             "set2": [],
             "expected": [1, 2, 3, 4]
          },
          {
-            "description": "no elements in common",
-            "set1": [1, 2, 3],
-            "set2": [4],
-            "expected": [1, 2, 3]
-         },
-         {
-            "description": "one element in common, one extra",
+            "description": "difference of two non-empty sets is a set of elements that are only in the first set",
             "set1": [3, 2, 1],
             "set2": [2, 4],
             "expected": [1, 3]
-         },
-         {
-            "description": "two elements in common, one extra",
-            "set1": [1, 2, 3, 4],
-            "set2": [3, 2, 5],
-            "expected": [1, 4]
          }
       ]
    },
-   "symmetric-difference": {
-      "description": [
-         "Produce the symmetric difference of two sets.  The symmetric",
-         "difference consists of elements in one or the other but not both."
-      ],
+   "equal": {
+      "description": "Sets with the same elements are equal",
       "cases": [
          {
-            "description": "two empty sets",
+            "description": "empty sets are equal",
             "set1": [],
             "set2": [],
-            "expected": []
+            "expected": true
          },
          {
-            "description": "empty set and non-empty set",
+            "description": "empty set is not equal to non-empty set",
             "set1": [],
-            "set2": [3, 2, 5],
-            "expected": [3, 2, 5]
+            "set2": [1, 2, 3],
+            "expected": false
          },
          {
-            "description": "non-empty set and empty set",
-            "set1": [1, 2, 3, 4],
+            "description": "non-empty set is not equal to empty set",
+            "set1": [1, 2, 3],
             "set2": [],
-            "expected": [1, 2, 3, 4]
+            "expected": false
          },
          {
-            "description": "no elements in common",
-            "set1": [1, 2, 3],
-            "set2": [4],
-            "expected": [1, 2, 3, 4]
+            "description": "sets with the same elements are equal",
+            "set1": [1,2],
```

Comment:
do we care very much about consistent spacing (space after comma)? If so, should deal with this and the other test a few lines below